citizen/citizen.c

```diff
@@ -379,18 +422,20 @@ int getCitizenAge(List *list, int citizenId) {
  * @param list A pointer to the list of citizens.
  * @return The number of male citizens in the list.
  */
-int getMalePopulation(List *list) {
-    Node *current = list->head;  // Start from the head of the list
-    int malePopulation = 0;      // Counter for male population
+int getmale_population(List *list)
```

Comment:
please me consistent, the previous uses camelCase on the function naming convention

Reply:
Done. Thanks.